lib/improver/tests/blending/weighted_blend/test_rationalise_blend_time_coords.py

```diff
@@ -68,8 +66,6 @@ def setUp(self):
         # make a cube list and merged cube containing the two model cubes, for
         # use in defining reference coordinates for tests below
         self.cubelist = iris.cube.CubeList([self.ukv_cube, self.enuk_cube])
-        self.cube = merge_cubes(
```

Comment:
I agree with these changes. This function modifies a cubelist in place, so we should never have been trying to merge cubes in the tests